dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -128,7 +128,7 @@ def get_applications_by_role(role)
 
   def application_params
     params.require(:application).permit(
-      :status, :notes, :regional_partner_filter, :response_scores, :locked
+      :status, :notes, :regional_partner_filter, :response_scores, :locked, :pd_workshop_id
```

Comment:
This is causing an [error](https://app.honeybadger.io/projects/3240/faults/36190571#notice-comments) in production:

> ActiveModel::UnknownAttributeError: unknown attribute 'pd_workshop_id' for Pd::Application::Facilitator1819Application.